samples/core/volume_ops/volumeop.py

```diff
@@ -36,7 +36,5 @@ def volumeop_basic(size):
         pvolumes={"/mnt": vop.volume}
     )
 
-
-if __name__ == "__main__":
-    import kfp.compiler as compiler
-    compiler.Compiler().compile(volumeop_basic, __file__ + ".tar.gz")
+if __name__ == '__main__':
```

Comment:
nit: Actually for top layer component two blank lines would be better

Reply:
Not sure I understand, leaving two blank lines above main?